src/main/java/mate/academy/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,20 @@
 package mate.academy.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
```

Comment:
for String use `@NotBlank`

```suggestion
    @NotBlank
```